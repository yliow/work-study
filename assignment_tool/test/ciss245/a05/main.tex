
\input{thispreamble.tex}
\newcommand\myincludetex[1]{\textbox{{\scriptsize \texttt{#1}}}

    \input{#1}
}

\newcommand\myincludesrc[1]{\textbox{{\scriptsize \texttt{#1}}}
    
    \VerbatimInput[fontsize=\footnotesize,frame=single]{#1}
}

Q1. \subimport*{a05q01/doc/}{q01.tex}
\newpage Q2. \subimport*{a05q02/doc/}{q02.tex}
\newpage Q3. \subimport*{a05q03/doc/}{q03.tex}
\newpage Q4. \subimport*{a05q04/doc/}{q04.tex}
\newpage Q5. \subimport*{a05q05/doc/}{q05.tex}
\newpage Q6. \subimport*{a05q06/doc/}{q06.tex}
\newpage Q7. \subimport*{a05q07/doc/}{q07.tex}
\newpage Q8. \subimport*{a05q08/doc/}{q08.tex}
\newpage Q9. \subimport*{a05q09/doc/}{q09.tex}
\newpage Q10. \subimport*{a05q10/doc/}{q10.tex}
\newpage Q11. \subimport*{a05q11/doc/}{q11.tex}

\input{thispostamble.tex}
    