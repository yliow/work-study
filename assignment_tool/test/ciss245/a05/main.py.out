
\begin{center}
\begin{tikzpicture}[>=triangle 60,scale=.6]
\begin{axis}[
width=5in, height=4in,
xmin=-5, xmax=5,
ymin=-3, ymax=5,
axis x line=middle,
axis y line=middle,
grid=major
]
\addplot[->] coordinates {(0,0) (2,3)};
\end{axis}
\end{tikzpicture}
\end{center}

